patches/server/0026-Prevent-tile-entity-and-entity-crashes.patch

```diff
@@ -18,8 +18,8 @@ index d29334b5dee6fe3f292f774442304b7dc6b0e2f0..d6ed061e36668ea1cbaee44c06575534
 -            entity.fillCrashReportCategory(crashreportsystemdetails);
 -            throw new ReportedException(crashreport);
 +            // Paper start - Prevent tile entity and entity crashes
-+            System.err.println("Entity threw exception at " + entity.level.getWorld().getName() + ":" + entity.getX() + "," + entity.getY() + "," + entity.getZ());
-+            throwable.printStackTrace();
++            final String msg = String.format("Entity threw exception at %s:%s,%s,%s", entity.level.getWorld().getName(), entity.getX(), entity.getY(), entity.getZ());
++            MinecraftServer.LOGGER.error(msg, throwable);
```

Comment:
msg could be inlined and therefore use the log4j `{}` syntax

Reply:
Yeah, had it doing that, however a later patch uses the string given to the logger for the exception event